### ISSUE TITLE
Hello friends I need code review (peer to peer review)

### ISSUE THREAD

**Issue by akhror-valiev** [akhror-valiev/capstone-project#2]
https://github.com/akhror-valiev/capstone-project/blob/0a072d183a9778888dba8640d32383f16701ffda/index.html#L1-L184

**Comment by jonathanmunamire** [comment]
@akhror-valiev; Your Work has a good structure
- For more Accessibility, It should be better to add on line 158 an alt message to the image

**Comment by akhror-valiev** [comment]
Thanks, @jonathanmunamire, it will be fixed soon.

**Comment by Victoria129** [comment]
I think the code looks good! Keep up the good work!

**Comment by Daniel-Kigozi** [comment]
The code looks fine it's clean and well written, thanks for the good work
